src/Lachain.Core/RPC/HTTP/Web3/AccountServiceWeb3.cs

```diff
@@ -39,8 +40,60 @@ public class AccountServiceWeb3 : JsonRpcService
         public string GetBalance(string address, string tag)
         {
             var addressUint160 = address.HexToUInt160();
-            var availableBalance = GetSnapshotByTag(tag)!.Balances.GetBalance(addressUint160);
-            return Web3DataFormatUtils.Web3Number(availableBalance.ToWei().ToUInt256());
+
+            if (tag == "pending")
+            {
+                // Get all transaction from pool
+                var txpool = this._transactionPool;
+                var transactions = txpool.Transactions;
+
+                List<TransactionReceipt> tx_Receipts = new List<TransactionReceipt>();
+                
+                foreach(var tx in transactions)
+                {
+                    tx_Receipts.Add(tx.Value);
+                }
+
+                // Sort on the basis of nonce
+                tx_Receipts = tx_Receipts.OrderBy(receipt => receipt, new ReceiptComparer()).ToList();
```

Comment:
Should we consider sorting after we have filterred the necessary transactions ? 
Pool may be very large, it's not a good idea to sort them in each api call.
On the other hand, pending transactions from a particular address will be very small.